package.json

```diff
@@ -42,6 +42,7 @@
     "@material/fab": "0.3.16",
     "@material/form-field": "0.2.14",
     "@material/icon-toggle": "0.1.19",
+    "@material/layout-grid": "^0.4.4",
```

Comment:
In order to avoid breakage, I recommend locking to a specific version rather than allowing npm to upgrade via `^`.

Reply:
Done!